### DISCUSSION TITLE
Hide non-junction fields in drawer

### DISCUSSION THREAD

**Discussion by kegesch** [directus/directus#19040] [Feature Requests]
### Summary

A possibility to hide the non-junction fields of a M2M relationship in the drawer (the fields of the related item) so that only juction-table specific fields are shown.

### Motivation

Upon clicking on an item of a M2M-relationship, the drawer opens to edit the fields of this junction item AND the related item of the other table. In certain use-cases (usually those where the table of the junction items mostly list junction fields only) the user could think that they only edit the data for this certain item, but they will edit the data of the related item. 
A possibility to hide the non-junction fields in the drawer (the fields of the related item) would be nice to prevent this confusion for the user. 

### Detailed Design

A boolean dropdown in the interface section in the edit drawer of the M2M relationship field with the name "Hide fields of related item / Hide non-junction fields". 

The value "hide" will cause that the fields of the related item / non-junction table fields are hidden in the drawer.
The value "show" is the default and will show the fields of the related item (as it is implemented right now).

### Must Have:
- Junction-Item drawer can be configured to hide fields of the related item / schema

### Drawbacks

None

### Alternatives

An alternative could be to configure fields that should be shown / hidden individually, but there is no fitting place where such an interface to do so could be added.

### Adoption Strategy

No breaking change, just an extension.

**Comment by rijkvanzanten** [comment]
Wouldn't this effectively be the same as using a O2M interface to just the junction collection then? 🤔

**Comment by fitu86** [comment]
What I do is create a Boolean type field called overview, when activated it shows the empty fields hidden by default.

![InShot_20230708_233450554](https://github.com/directus/directus/assets/16139127/e5b72070-ab9f-4c61-83cb-dca75ff471c2)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
